web_external/views/map/MapPanel.js

```diff
@@ -318,11 +319,34 @@ const MapPanel = Panel.extend({
             this.map.center(center);
         });
 
+        this.listenTo(events, 'm:draw-boundary-dataset', () => {
+            if (!this.drawDatasetLayer) {
+                var layer = this.map.createLayer('annotation', {
+                    annotations: ['rectangle'],
+                    showLabels: false
+                });
+                layer.geoOn(geo.event.annotation.state, (e) => {
+                    bootbox.prompt({
+                        title: 'Give dataset a name?',
```

Comment:
I would change this title to `Boundary dataset name` (or at least remove the question mark).

Reply:
Make sense.